### ISSUE TITLE
ClumsySyntax false positive

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#1008]
ClumsySyntax issue raised, "statement should end with a period".

```
NO-RETURN-VALUE wordAppl:AppHide("Microsoft Word").
```

**Comment by gquerret** [comment]
Issue coming from NoReturnValueStatement rule in the grammar.
Fixed in develop.
